test/extensions/filters/http/common/fuzz/uber_filter.cc

```diff
@@ -91,6 +93,11 @@ void UberFilterFuzzer::fuzz(
 
   // Data path should not throw exceptions.
   if (decoder_filter_ != nullptr) {
+    Http::TestRequestHeaderMapImpl headers;
```

Comment:
Probably not related to this PR, but I wonder if we need to do the same for the body and trailers (and also for the upstream side). This should be done in followup PR.

Reply:
ack